### ISSUE TITLE
Ignore unknown properties

### ISSUE THREAD

**Issue by jamiewinder** [NetTopologySuite/NetTopologySuite.IO.GeoJSON#85]
Hi. I'm using this library (the STJ version) with an API which unfortunate includes a non-standard property within the GeoJSON root element. This seems to throw the converter off and I get an error of 
```
The converter 'NetTopologySuite.IO.Converters.StjGeometryConverter' read too much or not enough.
```
Is it possible to get the converter to ignore these properties, or a workaround?

Here's an example of what the non-standard property (`type_string`) looks like:
```
{"type": "LineString", "type_string": "LineString", coordinates: []}
```

**Comment by oshawa-connection** [comment]
This isn't an issue specific to this library. You can ignore a field by adding the `System.Text.Json.Serialization.JsonIgnore` attribute to the field like this:
```csharp
    public class MyModel
    {

        [Key]
        public int id { get; set; }

        public NetTopologySuite.Geometries.Point geom { get; set; }

        public string somefield { get; set; }

        [System.Text.Json.Serialization.JsonIgnore]
        public string fieldToIgnore { get; set; }
```
Hopefully that helps?

**Comment by DGuidi** [comment]
fixed by #95
